9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
How about just populating a database with tables and data? User and db creation is already handled. It would be nice if the script used environment variables like MYSQL_USER, MYSQL_DATABASE and so on. They should be exported and available, though I don't think we ever tested this.